### PR TITLE
Update to v12 stuF (i don't think im done aaaa)

### PR DESCRIPTION
**Pull request by Naim2000**
Color roles not working, most likely because discord.js is updated to use managers (found in cache)
(and super additions)

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = {
 	},
 	
     "opID": "330499035419115522",
-    "opIDs": ["281134216115257344", "330499035419115522"], //TODO i dont need multiple ops anymore
+    "opIDs": ["281134216115257344", "330499035419115522"], //TODO i dont need multiple ops anymore // -_-
```
Comment (ledlamp) [inline src/config.js:17]: what

Comment (Naim2000) [reply src/config.js:17]: "todo"
is literally like what
22 characters to remove? 21? idk just do it

**Comment by ledlamp** [comment]
bruh some code uses it

**Comment by Naim2000** [comment]
yeah but what discord user uses it

**Comment by Naim2000** [comment]
wait why is the pull request closed

**Comment by Naim2000** [comment]
buh

**Pull request closed without merging**